presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/BenchmarkReorderInterconnectedJoins.java

```diff
@@ -76,7 +75,6 @@ public static class BenchmarkInfo
         @Setup
         public void setup()
         {
-            checkState(numberOfTables >= 2, "numberOfTables must be >= 2");
```

Comment:
This behavior assertion is good to have IMO (single table cannot logically join), even though the test will no-op for the <2 case